src/tests/__tests__/__snapshots__/Checkbox.react-test.js.snap

```diff
@@ -13,6 +13,7 @@ exports[`Checkbox Component render 1`] = `
         disabled={undefined}
         id="id1"
         name="checkbox"
+        onBlur={undefined}
```

Comment:
would be great to test these too but that's minor, don't want to postpone this even further